content/patterns/checkbox/examples/checkbox.html

```diff
@@ -18,101 +18,102 @@
-              <li><div role="checkbox" aria-checked="false" tabindex="0">Mustard</div></li>
-              <li><div role="checkbox" aria-checked="false" tabindex="0">Sprouts</div></li>
+              <li><div role="checkbox" aria-checked="false" tabindex="0">レタス</div></li>
+              <li><div role="checkbox" aria-checked="true" tabindex="0">トマト</div></li>
+              <li><div role="checkbox" aria-checked="false" tabindex="0">マスタード</div></li>
+              <li><div role="checkbox" aria-checked="false" tabindex="0">スプラウト</div></li>
             </ul>
           </div>
         </div>
 
-        <div role="separator" id="ex_end_sep" aria-labelledby="ex_end_sep ex_label" aria-label="End of"></div>
+        <div role="separator" id="ex_end_sep" aria-labelledby="ex_label ex_end_sep" aria-label="の終了"></div>
       </section>
 
       <section>
-        <h2>Accessibility Features</h2>
+        <h2>アクセシビリティ機能</h2>
         <ul>
-          <li>To help assistive technology users understand that each checkbox is part of a set of related checkboxes named <q>Sandwich Condiments</q>, the checkboxes are wrapped in a <code>group</code> labeled by the <code>h3</code> heading element.</li>
-          <li>To enable assistive technology users to perceive the set of checkboxes as a list of four items, each <code>div</code> element that serves as a checkbox is contained within a <code>li</code> element contained by a <code>ul</code> element.</li>
-          <li>To make it easier to perceive that clicking either the label or checkbox will activate the checkbox, when a pointer hovers over either the checkbox or label, the background color changes, a border appears, and the cursor changes to a pointer.</li>
+          <li>支援技術の利用者が各チェックボックスが「サンドイッチの調味料」という名前の関連チェックボックスのセットの一部であることを理解するのに役立つように、チェックボックスは <code>h3</code> 見出し要素によってラベル付けされた <code>group</code> にラップされています。</li>
+          <li>支援技術の利用者がチェックボックスのセットを4つの項目のリストとして認識できるように、チェックボックスとして機能する各 <code>div</code> 要素は、<code>ul</code> 要素によって含まれる <code>li</code> 要素内に含まれています。</li>
+          <li>ラベル又はチェックボックスのいずれかをクリックするとチェックボックスがアクティブになることを知覚しやすくするために、ポインターがチェックボックス又はラベルのいずれかにホバーすると、背景色が変わり、境界線が表示され、カーソルがポインターに変わります。</li>
           <li>
-            Because transparent borders are visible on some systems when operating system high contrast settings are enabled, transparency cannot be used to create a visual difference between the element that is focused and other elements.
-            Instead of using transparency, the focused element has a thicker border and less padding.
-            When an element receives focus, its border changes from 0 to 2 pixels and padding is reduced by 2 pixels.
-            When an element loses focus, its border changes from 2 pixels to 0 and padding is increased by 2 pixels.
+            オペレーティングシステムのハイコントラスト設定が有効になっている場合、一部のシステムでは透明な境界線が表示されるため、フォーカスされている要素と他の要素との間に視覚的な違いを作成するために透明度を使用することはできません。
```

Comment:
https://github.com/waic/aria-practices/pull/158/files#r2441692124 と同じ


```suggestion
            オペレーティングシステムのハイコントラスト設定が有効になっている場合、一部のシステムでは透明に設定された境界線が可視化されるため、フォーカスされている要素と他の要素との間に視覚的な違いを作成するために透明度を使用することはできません。
```